{
Kd: 0.0,
Kff: 0.0,
Kp: 1.5,
always_show_protective_laser: false, #false - will only show laser when obstacle detected
back_laser_link: hokuyo_back_laser_link,
base_frame_id: base_footprint,
deaccel_distance_thresh: 0.18,
debug: true,
delta_angle_stuck_th: 5,
erro_theta2_stuck_th_low: 80,
erro_theta2_stuck_th_sup: 100,
error_deviation_deg: 5.0,
#forward_protective_distance: 1.2,
#forward_warning_distance: 1.5,
front_laser_link: hokuyo_front_laser_link,
global_frame_id: map,
group_points: false,
k_r: 0.0,
keep_ratio: true,
#lateral_protective_distance: 0.3,
#lateral_warning_distance: 0.3,
limit_velocity: true,
max_angular_vel: 0.4,
max_angular_vel_hard: 999.0,
max_linear_vel: 0.3,
min_angular_vel: 0.02,
min_linear_vel: 0.03,
next_target_timeout: 20,
obstacle_detection: true,
obstacle_timeout: 20,
plan_topic: /oea_planner/oea_plan,
scale_on_curves: 0.3,
send_markers: true,
stop_with_joystick: true,
timeout: 5,
timer_rate: 100,
tolerance_d: 0.1,
tolerance_d_last_point: 0.05,
tolerance_erro_theta2: 85,
tolerance_yaw_deg: 15.0,
tolerance_yaw_deg_last_point: 2.5,
use_yaw_on_middle_points: true,
safety_area: 
{laser_protective_fwd_dist: 1.0,
 laser_protective_lat_dist: 0.5,
 laser_warning_fwd_dist: 2.0,
 laser_warning_lat_dist: 0.8,
 outlier_protective_fwd_dist: 1.0,
 outlier_protective_lat_dist: 1.0,
 outlier_warning_fwd_dist: 1.5,
 outlier_warning_lat_dist: 1.5
}
}
